src/configuration.ts

```diff
@@ -20,9 +21,7 @@ export interface ConfigLoaderOptions {
 }
 
 export function load(options: ConfigLoaderOptions = {}): Configuration {
-  let cwd = process.cwd();
-  let rootPath = execa.sync("git", ["rev-parse", "--show-toplevel"], { cwd }).stdout;
```

Comment:
Not really related to this PR, but I figured the git command belongs to the `git` file.